baselines/common/cmd_util.py

```diff
@@ -187,7 +166,7 @@ def robotics_arg_parser():
 
 def parse_unknown_args(args):
     """
-    Parse arguments not consumed by arg parser into a dictionary
+    Parse arguments not consumed by arg parser into a dicitonary
```

Comment:
I like old spelling better :)

Reply:
Aha, turns out it is changed in a recent pr, but I didn't sync from it.
https://github.com/openai/baselines/pull/917/files